related_projects/entry.json

```diff
@@ -0,0 +1,14 @@
+[{
+  "name": {
+    "en" : "Related projects"
+  },
+  "description": {
+    "en": "Agregate to the community extensions in order to directly link the contracts with their respective projects (when applies)"
+  },
```

Comment:
Would suggest to update the description to: "Adds a related projects array to allow for the contracting process to be linked to its respective projects."